source/containers/Form/hooks/formActions.js

```diff
@@ -59,3 +63,16 @@ export function increaseFormCounter(state) {
     counter: increaseCount(counter, steps.length),
   };
 }
+/**
+ * Action to run when starting a form.
+ * @param {object} state the current state of the form
+ */
+export function startForm(state, payload) {
+  // TODO: Pass user input values.
+  payload.callback({});
+  const { steps, counter } = state;
+  return {
```

Comment:
It's not a issue, but i agree that it might be a bit confusing. Might be a good idea to pass the callback to the increaseCounter function instead and avoiding this behavior :)

Reply:
Since this PR is currently is a show blocker, I suggest that we look over this behavior in a future sprint :)

Reply:
Yaa can do that ..  :)